data-syncs/getting-started-with-data-syncs.md

```diff
@@ -1,102 +1,86 @@
 {% endhint %}
 
-#### Connections options
-
-You can choose whether to sync data one-way in, one way out, or bi-directionally into your Data Network.
+### 3. **Choose connection options**
 
-For more information on the different types of connections in Cinchy, review the following pages:
+Decide on your data sync direction:
+* One-way in
+* One-way out
+* Bi-directional
 
+Explore various connection types:
 * [Supported data sync sources](supported-data-sync-sources/)
 * [Supported data sync destinations](supported-data-sync-destinations/)
 * [Supported sync stream sources (for real-time syncs)](supported-real-time-sync-stream-sources/)
 
-### Create a data sync configuration
-
-Whether you are setting up a real-time or a batch sync, you will need to create your data sync configuration. The data sync configuration defines the source, the mapping to the target, and synchronization behavior.
-
-To set up a data sync, you can use the Connections UI or manually input a Data Sync XML into the Data Sync Configuration table. For more information, [see the Building data syncs page.](../data-syncs/building-data-syncs/README)
-
-### Set up a listener config (real-time syncs)
-
-If you are setting up a real-time sync, you will need to set up your listener configuration in the Listener Config table.
+### 4. **Create a data sync configuration**
 
-For instructions about subscribing to your event stream, [see the supported real-time sync stream sources page](supported-real-time-sync-stream-sources/)
+For both real-time and batch syncs, you'll need to set up your configuration:
 
-### Test and run
+* Define the source
+* Map to the target
+* Determine synchronization behavior
 
-After you create and add the data sync config to the table in Cinchy, you can test your data sync:
+Use the Connections UI or manually input a Data Sync XML into the Data Sync Configuration table. More details are on the [Building data syncs page.](../data-syncs/building-data-syncs/README)
 
-* **For batch syncs:** run your job in the Connections UI.
-* **For real-time syncs:** You can test by making changes in the source system and seeing if the target updates.
+### 5. **Set up Listener Config (For Real-time Syncs)**
 
-## Schedule a data sync
+For real-time syncs, configure your listener in the **Listener Config** table. [See the supported real-time sync stream sources page](supported-real-time-sync-stream-sources/) for subscribing to your event stream.
 
-You can set up a schedule for batch syncs using a task/job scheduling application.
+### 6. **Test and Run**
 
-<!-- 
+After setting up the configuration:
+* **Batch syncs:** Execute your job through the Connections UI.
+* **Real-time syncs:** Test by altering the source system and checking if the target updates.
 
-Need to check with SMEs on page update.
+### 7. **Schedule Your Data Sync**
 
-## Monitor a data sync
+Batch syncs can be scheduled using a task/job scheduling application.
```

Comment:
Would be good to note how.  There's the CLI syncdata command that can be scheduled in an external script

Reply:
This section needs a topic. I'll try and schedule it after I finish looking at where to put the Connections Functions for @ID and @Column